pkg/cli/dump.go

```diff
@@ -88,10 +88,12 @@ func dumpTable(w io.Writer, conn *sqlConn, origDBName, origTableName string) err
 	if err != nil {
 		return err
 	}
-	vals = make([]driver.Value, 2)
+	defer func() { _ = rows.Close() }()
```

Comment:
you're calling `Close` twice on this instance of `rows`, in the defer and at L113.
While it's probably no big deal, it feels like this part of the function could be moved to a helper function.